src/main/java/org/kuali/kfs/pdp/batch/service/impl/Iso20022FormatExtractor.java

```diff
@@ -833,8 +845,15 @@ private PostalAddress6 constructPostalAddress(
         String isoCountryCode;
         if (StringUtils.isNotBlank(countryCode)) {
             isoCountryCode = convertFIPSCountryCodeToISOCountryCode(countryCode);
+        } else if (stateAndPostalCodeAreWithinUnitedStates(templateCustomerProfile.getStateCode(),
+                templateCustomerProfile.getZipCode())) {
+            LOG.warn("constructPostalAddress, Customer Profile {} has a blank country code but seems to reference "
+                    + "a domestic address; defaulting to '{}'",
+                    templateCustomerProfile.getId(), KFSConstants.COUNTRY_CODE_UNITED_STATES);
+            isoCountryCode = KFSConstants.COUNTRY_CODE_UNITED_STATES;
         } else {
-            LOG.warn("constructPostalAddress, Customer Profile {} has a blank country code; defaulting to '{}'",
+            LOG.warn("constructPostalAddress, Customer Profile {} has a blank country code but seems to reference "
+                    + "a foreign address; defaulting to '{}'",
```

Comment:
Suggestion: "defaulting to" => "defaulting iso country code to"

Reply:
I'll update the wording accordingly. I think the phrase "defaulting to ISO country code" would work a little better; please let me know if you would prefer your suggested wording instead.